lib/packages/core/lib/src/blocs/conversation/conversation_bloc.dart

```diff
@@ -76,9 +76,10 @@ class LMChatConversationBloc
       (event) {
         if (event.snapshot.value != null && _currentChatroomId != null) {
           final response = event.snapshot.value as Map;
-          final conversationId = int.parse(response["collabcard"]["answer_id"]);
+          final conversationId = int.tryParse(response["answer_id"]);
```

Comment:
this is changed from backend? or we missed it in our previous release?

Reply:
No no, this has changed because of how I am parsing from FCM now. Nothing to do with backend.